docs/assets/js/script.js

```diff
@@ -11,7 +11,7 @@ devicon.controller('IconListCtrl', function($scope, $http, $compile) {
   // Determination of the latest release tagging
   // which is used for showing in the header of the page
   // as well as for CDN links
-  var gitHubPath = 'devicons/devicon';
+  var gitHubPath = 'snailedlt/devicon';
```

Comment:
This should probably not be changed

Reply:
nice catch. Yes definitely should be changed

Reply:
fixed: 726d58f1bcadbd4da6910e041d0bf5a5d3ec092b